components/AppFooter.vue

```diff
@@ -64,4 +72,38 @@
       text-decoration: none;
     }
   }
+
+  .footer-menu {
+    list-style: none;
+    margin: 2em 0 0;
+    padding-left: 0;
+  }
+
+  .footer-menu li {
+    display: inline-block;
+    margin: 0 2em 1em 0;
+  }
+
+  .footer-menu a {
+    border-bottom: 1px solid transparent;
+    color: #666;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    @media print {
+      text-transform: unset;
+    }
+
+    &:hover,
+    &:focus {
+      text-decoration: underline;
+    }
+
+    @media print {
+      &:after {
+        content:" <" attr(href) "> ";
+      }
+    }
```

Comment:
Thanks for doing the extra work to follow conventions! I think it's good to expose URLs on a printed sheet. Your effort will probably be rewarded with a refactor in January to remove all printed URLs because most offices dislike them 😢

Reply:
Any chance we can leave this in but give a toggle in Contentful per country? I personally think the URLs are *genius* if you're actually printing the PDFs... the only mod I'd make is to use (brackets) instead of &lt;angle brackets>.

Reply:
Toggling per country can be achieved, but it's not a simple change and will introduce a whole new concept of configuration instead of just managing content in Contentful.

Not a problem, but also worth discussing before we dive in and add UI widgets.

We can update the chevrons to parens no problem, but I was following multiple "printing best bractices" guides by using chevrons. It's more traditional or something, who knows.

Reply:
https://humanitarian.atlassian.net/browse/DSR-127